iaso/api/profiles.py

```diff
@@ -198,9 +198,11 @@ def partial_update(self, request, pk=None):
 
         if not username:
             return JsonResponse({"errorKey": "user_name", "errorMessage": _("Nom d'utilisateur requis")}, status=400)
-        existing_user = User.objects.get(username__iexact=username)
+
         user = profile.user
-        if existing_user and not existing_user.id == user.id:
+        existing_user = User.objects.filter(username__iexact=username).filter(~Q(pk=user.id))
```

Comment:
In fact, i discover now that if you update the username in the same request it crashes the api, we should find a better way to find existing user